### ISSUE TITLE
Is this project still alive?

### ISSUE THREAD

**Issue by braph** [UraniumDonut/nbfc-revive#130]
Just wondering ...

**Comment by abb3v** [comment]
PRs still seem to be coming through. My guess is maintainer doesn't want to deal with approving the load anymore.

**Comment by UraniumDonut** [comment]
Yeah, I'm a little burned out. I'll get around to building it again. If there are some other reputable maintainers who want to join, I'm open for it!

**Comment by abb3v** [comment]
@UraniumDonut at what point should we be expecting another release?
